.github/pull_request_template.md

```diff
@@ -1,12 +1,9 @@
-## Why was this change made?
+## Why was this change made? 🤔
 
 
 
-## How was this change tested?
-
-
-
-## Which documentation and/or configurations were updated?
+## How was this change tested? 🤨
 
+⚡ ⚠ If this change has cross service impact, ***run [integration test create_preassembly_image_spec.rb](https://github.com/sul-dlss/infrastructure-integration-test) on stage as it tests preservation*** and/or test in stage environment, in addition to specs. ⚡
```

Comment:
```suggestion
⚡ ⚠ If this change has cross service impact, or if it changes [code used in the replication pipeline](https://github.com/sul-dlss/preservation_catalog/blob/main/app/jobs/README.md), ***run [integration test create_preassembly_image_spec.rb](https://github.com/sul-dlss/infrastructure-integration-test) on stage as it tests preservation*** and/or test in stage environment, in addition to specs. ⚡
```

Reply:
ha, merged, will do a small follow on 🙂

Reply:
@jmartin-sul beware your editor removing the desired blank lines at the end, and maybe refer to "jobs" and other specifics rather than "replication pipeline" for those less familiar with the code base.